tests/telemetry/events/swss_events.py

```diff
@@ -26,6 +26,9 @@
 
 
 def test_event(duthost, gnxi_path, ptfhost, data_dir, validate_yang):
+    if duthost.topo_type.lower() in ["m0", "mx", "m0-2vlan"]:
```

Comment:
Testbed type `m0-2vlan` is not supported yet:

Please refer to tests/common/testbed.py
```
    def get_testbed_type(self, topo_name):
        pattern = re.compile(r'^(wan|t0|t1|ptf|fullmesh|dualtor|t2|tgen|mgmttor|m0|mc0|mx|dpu)')
```

Reply:
Thanks for pointing out! Removed this topo type.